aws/vars.tf

```diff
@@ -4,6 +4,12 @@ variable "aws_region" {
   default     = "us-west-2"
 }
 
+variable "availability_zone" {
+  description = "The AWS AZ to deploy the infrastructure on"
+  default     = ["us-west-2a", "us-west-2b", "us-west-2c"]
```

Comment:
it might be nice to default to single az

Reply:
and add a comment about the cost impact of enabling multi-az